e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+print_prefix="E2E: STREAM MODE TEST:"
+echo "${print_prefix} START OF STREAM MODE TEST"
+echo "${print_prefix} STARTING OPENMRS SERVER"
+docker-compose -f openmrs-compose.yaml up -d --remove-orphans
+openmrs_start_wait_time=0
+contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 \
+ --max-time 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null | cut -d ";" -f 1)
+ until [[ ${contenttype} == "application/json" ]]; do
+      sleep 60s;
+      echo "${print_prefix} WAITING FOR OPENMRS SERVER TO START"
+      contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 \
+       --max-time 20 http://localhost:8099/openmrs/ws/rest/v1/visittype 2>/dev/null | cut -d ";" -f 1)
+      ((openmrs_start_wait_time+=1))
+      if [[ ${openmrs_start_wait_time} == 20 ]]; then
+        echo "${print_prefix} TERMINATING TEST AS OPENMRS TOOK TOO LONG TO START"
+        exit 1
+      fi
+done
+echo "${print_prefix} OPENMRS SERVER STARTED SUCCESSFULLY"
+echo "${print_prefix} BUILDING THE ANALYTICS PROJECT"
+test_dir=$(mktemp -d -t analytics_tests__XXXXXX)
+echo "${print_prefix} PARQUET FILES WILL BE WRITTEN INTO $test_dir DIRECTORY"
+echo "${print_prefix} RUNNING STREAM MODE WITH PARQUET SYNC"
+echo "${print_prefix}mvn compile exec:java -pl streaming-binlog \
+   -Dexec.args= --databaseHostName=localhost \
+   --databaseUser=root --databasePassword=debezium \
+   --databaseSchema=openmrs \
+   --openmrsUserName=admin --openmrsPassword=Admin123 \
+   --openmrsServerUrl=http://localhost:8099/openmrs \
+   --openmrsfhirBaseEndpoint=/ws/fhir2/R4 \
+   --snapshotMode=initial \
+   --fhirSinkPath='${test_dir}' \"
+
+mvn compile exec:java -pl streaming-binlog \
+   -Dexec.args="--databaseHostName=localhost \
+   --databaseUser=root --databasePassword=debezium \
```

Comment:
@gitcliff , you actually already moved debzium specific settings to a json config file